test_removal_reasons.py

```
@@ -0,0 +1,178 @@
+Test script to verify removal reason processing without Reddit API calls
+Creates a local markdown file to demonstrate the functionality
+"""
+import sqlite3
+from datetime import datetime, timezone
+import os
+import sys
+
+# Add the current directory to path to import our module
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+from modlog_wiki_publisher import *
```

Ruff (0.15.12)

[error] `from modlog_wiki_publisher import *` used; unable to detect undefined names

(F403)